release-doc-in-process/all_vf.TTL

```diff
@@ -339,6 +339,13 @@ vf:containedIn a            owl:ObjectProperty ;
         vs:term_status      "unstable" ;
         rdfs:comment        "Used when a stock economic resource contains units also defined as economic resources." .
 
+vf:primaryAccountable a      owl:ObjectProperty ;
+        rdfs:label          "primary accountable" ;
+        rdfs:domain         vf:EconomicResource ;
+        rdfs:range          vf:Agent ;
+        vs:term_status      "unstable" ;
+        rdfs:comment        "The agent currently with primary rights and responsibilites for the economic resource. This agent will normally do the accounting for the resource." .
```

Comment:
The description in the overview of this PR is more clear. Suggest changing
> This agent will normally do the accounting for the resource.

to
>  This is the agent that includes the resource in their accounting reports and/or as part of their inventory.

Or something like that...maybe mention accountingQuantity? "Do the accounting" could mean "be the bookkeeper or accountant".

Reply:
@bhaugen please check the updated definition.